### PR TITLE
￼ Several changes (add support for MUON Run3 data model, removed TaskName, etc.)

### PR DESCRIPTION
**Pull request by iarsene**
ReducedInfoTable:
* removed nuclei PID columns
* added few other columns (TRDpattern, event timestamp)
* changed name of ReducedMuonExtended table to ReducedMuonExtra
* prepared new tables to support the new MUON Run3 data model (commented out for now)

CutsLibrary:
* several cuts added for the study of the TPC dE/dx degradation impact on quarkonia triggers in high lumi pp

HistogramsLibrary
* updated trigger histograms
* added more PID histograms
* Run3 MUON histograms (commented out for now)

VarManager
* added more variables (trigger, PID, Run3 MUON)
* replaced some TMath expressions with std

filterPP:
* task names changed to avoid conflicts
* Removed TaskName in the workflow config

tableMaker
* added changes for Run3 muon
* Filter applied now on Run2Track to reject tracklets
* implemented changes from data model

Other tasks:
small changes to remove TaskName where not needed

### CODE REVIEW

File: Analysis/PWGDQ/include/PWGDQCore/VarManager.h [hunk 1]
```diff
@@ -74,15 +80,20 @@ class VarManager : public TObject
     kNRunWiseVariables,
 
     // Event wise variables   // Daria: imedjat ai ziua mja
```
Comment (jgrosseo) [inline Analysis/PWGDQ/include/PWGDQCore/VarManager.h]: :)

File: Analysis/Tasks/PWGDQ/dileptonMuMu.cxx [hunk 2]
```diff
@@ -71,7 +70,7 @@ void DefineHistograms(HistogramManager* histMan, TString histClasses);
 //        This is a temporary fix until the arrow/ROOT issues are solved, at which point it will be possible
 //           to automatically detect the object types transmitted to the VarManager
 constexpr static uint32_t gkEventFillMap = VarManager::ObjTypes::ReducedEvent | VarManager::ObjTypes::ReducedEventExtended;
-constexpr static uint32_t gkMuonFillMap = VarManager::ObjTypes::ReducedTrack | VarManager::ObjTypes::ReducedTrackMuon;
+constexpr static uint32_t gkMuonFillMap = VarManager::ObjTypes::ReducedTrack | VarManager::ObjTypes::ReducedMuon;
 
 struct EventSelection {
```
Comment (jgrosseo) [inline Analysis/Tasks/PWGDQ/dileptonMuMu.cxx]: You don't want to rename this one as well?

File: Analysis/Tasks/PWGDQ/tableMaker.cxx [hunk 3]
```diff
@@ -48,6 +48,9 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 using MyBarrelTracks = soa::Join<aod::Tracks, aod::TracksExtra, aod::TracksCov, aod::TracksExtended, aod::TrackSelection, aod::pidRespTPC, aod::pidRespTOF, aod::pidRespTOFbeta>;
```
Comment (jgrosseo) [inline Analysis/Tasks/PWGDQ/tableMaker.cxx:48]: Could you already subscribe to the split TPC PID tables? Either tiny or split?

File: Analysis/PWGDQ/include/PWGDQCore/VarManager.h (hunk 1 above)
Comment (iarsene) [reply Analysis/PWGDQ/include/PWGDQCore/VarManager.h]: sorry :)

Comment (iarsene) [reply Analysis/PWGDQ/include/PWGDQCore/VarManager.h]: fixed now

File: Analysis/Tasks/PWGDQ/dileptonMuMu.cxx (hunk 2 above)
Comment (iarsene) [reply Analysis/Tasks/PWGDQ/dileptonMuMu.cxx]: yes, i went through all of the workflows and added "DQ" in all task names, with a few exceptions, which have very particular names

File: Analysis/Tasks/PWGDQ/tableMaker.cxx (hunk 3 above)
Comment (iarsene) [reply Analysis/Tasks/PWGDQ/tableMaker.cxx:48]: I am planning to do this. In fact i am discussing now with Nicolo regarding the tiny version. I will ask one of the people in our group to run a check with the tiny version

**Comment by iarsene** [comment]
As i discussed it with @njacazio, i will prepare another PR adding the split tables. I would like to merge this PR in order to have it available for a hyperloop train tomorrow.

**Comment by iarsene** [comment]
The fullCI crash is unrelated, merging

**Pull request merged**